packages/core/tests/unit/serializers/metric-label-test.ts

```diff
@@ -22,20 +23,22 @@ module('Unit | Serializer | metric label', function(hooks) {
 
   test('normalizeMetricLabelV2', function(this: TestContext, assert) {
     const store = this.owner.lookup('service:store') as StoreService;
-    const request = store.createFragment('bard-request-v2/request', {
-      table: 'tableName',
-      columns: [
-        {
-          type: 'metric',
-          cid: 'cid_rupees'
-        }
-      ],
-      filters: [],
-      sorts: [],
-      limit: null,
-      dataSource: 'bardOne',
-      requestVersion: '2.0'
-    });
+    const request = store
+      .createFragment('bard-request-v2/request', {
+        table: 'tableName',
+        columns: [
+          {
+            type: 'metric',
+            cid: 'cid_rupees'
+          }
+        ],
+        filters: [],
+        sorts: [],
+        limit: null,
+        dataSource: 'bardOne',
+        requestVersion: '2.0'
+      })
+      .toJSON() as RequestV2;
```

Comment:
```suggestion
      .toJSON() as RequestFragment;
```
RequestV2 is just the POJO form right?

Reply:
Yes but we're calling `.toJSON()` so it's accurate

Reply:
Oh right, but then why do we need to create the fragment? Couldn't we just use the payload as RequestV2?